infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -44,6 +44,21 @@
      */
     private static final Duration RECOVERY_RETRY_INTERVAL = Duration.ofSeconds(1);
 
+    /**
+     * Locally collected server metrics polling interval.
+     */
+    private static final Duration METRICS_POLL_INTERVAL = Duration.ofSeconds(3);
```

Comment:
please make it not constant

Reply:
This is your code, I didn't change, just moved up several lines.